qhana_plugin_runner/templates/simple_template.html

```diff
@@ -31,6 +31,9 @@
             {% if done %}
             {{ forms.submit("done", action=done)}}
             {% endif %}
+            {% if errors == {} and valid %}
+            <mat-icon aria-hidden="true" class="material-icon notranslate material-icons ng-star-inserted qhana-valid-icon">done</mat-icon>
```

Comment:
This actually works? A better solution would be to inline the icon SVG. Also make sure that the valid indicator has at least some text that can be seen (and read by screenreaders) explaining what the green checkmark actually means.